cves/kernel/CVE-2018-12232.yml

```
@@ -190,8 +195,9 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    SOCKET network access protocol
```

Maybe go into a little more detail about what this access protocol is? Where you would find it, what it typically entails, etc. (Just another sentence or two would be good)

---

Thank you! I have added this: 

```suggestion
    SOCKET Network access protocol, an implementation of a controller area network, a way for devices to communicate with each other without a host computer 
```